api/urls.py

```diff
@@ -9,25 +9,37 @@
     LanguageAndLevelViewSet,
     NonTeachingHelpViewSet,
     PersonalInfoViewSet,
+    PublicStudentViewSet,
     StudentViewSet,
     TeacherUnder18ViewSet,
     TeacherViewSet,
 )
 
-router = DefaultRouter()
-router.register(r"personal_info", PersonalInfoViewSet, basename="personal_info")
-router.register(r"students", StudentViewSet, basename="students")
-router.register(r"teachers", TeacherViewSet, basename="teachers")
-router.register(r"teachers_under_18", TeacherUnder18ViewSet, basename="teachers_under_18")
-router.register(r"age_ranges", AgeRangeViewSet, basename="age_ranges")
-router.register(r"languages_and_levels", LanguageAndLevelViewSet, basename="languages_and_levels")
-router.register(r"day_and_time_slots", DayAndTimeSlotViewSet, basename="day_and_time_slots")
-router.register(r"enrollment_test", EnrollmentTestViewSet, basename="enrollment_test")
-router.register(
+internal_router = DefaultRouter()
+internal_router.register(r"personal_info", PersonalInfoViewSet, basename="personal_info")
+internal_router.register(r"students", StudentViewSet, basename="students")
+internal_router.register(r"teachers", TeacherViewSet, basename="teachers")
+internal_router.register(r"teachers_under_18", TeacherUnder18ViewSet, basename="teachers_under_18")
+internal_router.register(r"age_ranges", AgeRangeViewSet, basename="age_ranges")
+internal_router.register(
+    r"languages_and_levels", LanguageAndLevelViewSet, basename="languages_and_levels"
+)
+internal_router.register(
+    r"day_and_time_slots", DayAndTimeSlotViewSet, basename="day_and_time_slots"
+)
+internal_router.register(r"enrollment_test", EnrollmentTestViewSet, basename="enrollment_test")
+internal_router.register(
     r"enrollment_test_result", EnrollmentTestResultViewSet, basename="enrollment_test_result"
 )
-router.register(r"non_teaching_help", NonTeachingHelpViewSet, basename="non_teaching_help")
+internal_router.register(
+    r"non_teaching_help", NonTeachingHelpViewSet, basename="non_teaching_help"
+)
+
+public_router = DefaultRouter()
```

Comment:
add a short comment explaining what "public" is? (Like with the class above)
Just so that someone reading this code doesn't think we actually have some *publicly* available data

Reply:
Done